docs/ops/matrix/MatMul_1.md

```diff
@@ -12,13 +12,15 @@
 
 Before matrix multiplication, there is an implicit shape alignment for input arguments. It consists of the following steps:
 
-1. If rank of an input less than 2 it is unsqueezed to 2D tensor by adding axes with size 1 to the left of the shape. For example, if input has shape `[S]` it will be reshaped to `[1, S]`. It is applied for each input independently.
+1. One-dimensional tensors unsqueezing is applied for each input independently. The axes inserted in this step will be removed from the output shape.
```

Comment:
What about 0-d inputs?

Reply:
How can we identify "axes inserted" in the output shape?

Reply:
Oh, we don't allow such inputs. Closed.

Reply:
0D inputs (scalars) are not allowed in the numpy matmul, we have such a restriction in the current openvino MatMul spec too:

> **Inputs**:
> 
> *   **1**: Input batch of matrices A. Rank >= 1. Required.
> 
> *   **2**: Input batch of matrices B. Rank >= 1. Required.